formatTest/oprintTests/expected_output/inlineRecord.re

```diff
@@ -0,0 +1,6 @@
+type t0 = T0 { t0: int, };
+type t1 = A { x: int, } | B | C { c1: string, c2: string, };
```

Comment:
Doesn't Reason have parens around constructor args?

Reply:
I don't think so. Here's an rtop session:

```rust
Reason # type a = | A {x:int}: a;
type a = A(): a;
Reason # type a = | A {x:int};
type a = A();
Reason # type a = | A({x:int});;
Error: Syntax error
Reason # type a = | A {x:int}: a;
type a = A(): a;
Reason # type a = | A ({x:int}): a;
Error: Syntax error
```

Reply:
Maybe it should, though?

Reply:
The lack of support for wrapping parens for inline records, when they're required in other cases, is very confusing.  It's an inconsistency which would be good to fix.

Reply:
Yeah, I just saw someone get pretty confused about this too.
One somewhat related question I have is if there might ever be a change in OCaml that requires syntactically distinguishing values of inline variant records vs. values of variants that happen to contain records as their payload. I doubt it. That tells me that we should probably move to consistent parens everywhere.
If we hypothetically one day figured out a way to be able to drop parens for single argument things, then we would apply it consistently everywhere, not just on variants. Good discussion, thanks.